components/context/ThemeContext.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import React, { createContext, useState, useContext, ReactNode } from 'react';
+// eslint-disable-next-line import/no-unresolved
+
+interface ThemeContextType {
+  theme: 'day' | 'night';
+  toggleTheme: () => void;
+}
+
+const ThemeContext = createContext<ThemeContextType | undefined>(undefined);
+
+export const ThemeProvider: React.FC<{ children: ReactNode }> = ({ children }) => {
+  const [theme, setTheme] = useState<'day' | 'night'>('day');
```

Comment:
the default theme should be dark.
please rename this to be light and dark, with dark being the default

Reply:
Also we might need to use localStorage/cookies + useEffect to achieve this.